samples/wasm-c-api/CMakeLists.txt

```diff
@@ -87,6 +87,16 @@ endif()
 set(WAMR_ROOT_DIR ${CMAKE_CURRENT_LIST_DIR}/../..)
 include (${WAMR_ROOT_DIR}/build-scripts/runtime_lib.cmake)
 
+if (NOT DEFINED SANITIZER)
+  set(SANITIZER "")
+endif()
+
+if (SANITIZER STREQUAL "ubsan")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -O2 -fno-omit-frame-pointer -fsanitize=undefined -fno-sanitize-recover=all -fno-sanitize=alignment" )
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=undefined")
+endif()
```

Comment:
could we possibly add an `else` branch to warn about unsupported / invalid sanitizer?

Reply:
yeah, we probably can. 

I mean, I'm not sure it's harmful in any way, but let's add for easier debug in case of any errors